plugin/src/main/scala/com/stripe/sbt/bazel/BazelAst.scala

```diff
@@ -8,7 +8,15 @@ object BazelAst {
 
   final case class PyCall(
     funName: String,
-    args: List[(String, PyExpr)]
+    args: List[PyExpr] = List(),
```

Comment:
nit: `Nil` is slightly more efficient `List()` is a var-args so it allocates a zero length array for these.